test/e2e/specs/specs-wpcom/wp-calypso-gutenberg-page-editor-tracking-spec.js

```diff
@@ -40,10 +40,16 @@ describe( `[${ host }] Calypso Gutenberg Page Editor Tracking: (${ screenSize })
 			await wpAdminSidebar.selectNewPage();
 
 			const editor = await GutenbergEditorComponent.Expect( this.driver, 'wp-admin' );
-			await editor.initEditor( { dismissPageTemplateLocator: false } );
+			await editor.initEditor( { dismissPageTemplateLocator: true } );
 		} );
 
 		it( 'Tracks "from_template_selector" property for "wpcom_block_inserted"', async function () {
+			// Reload for page layout selector.
+			await this.driver.navigate().refresh();
+			await driverHelper.dismissAlertIfPresent( this.driver );
+			const editor = await GutenbergEditorComponent.Expect( this.driver, 'wp-admin' );
+			await editor.initEditor();
```

Comment:
Move this to the end of this function to make sure the next test starts with a clean state. Also add `{ dismissPageTemplateLocator: true }`

Reply:
Move this? or duplicate the process of refresh/init at the end?